Makefile

```
@@ -120,14 +121,18 @@ define PRINT_KEYBOARD
 	$(info $(PRINTING_KEYBOARD))
 endef
 
+.PHONY: generate-keyboards-file
 generate-keyboards-file:
 	$(foreach PRINTING_KEYBOARD,$(KEYBOARDS),$(eval $(call PRINT_KEYBOARD)))
 	exit 0
 
+.PHONY: clean
 clean:
-	echo -n 'Deleting .build ... '
+	echo 'Deleting .build/ ...'
 	rm -rf $(BUILD_DIR)
-	echo 'done'
+	echo 'Deleting *.bin and *.hex ...'
+	rm -f *.bin *.hex
+	echo 'Done.'
 	exit 0
 
```

```suggestion

.PHONY: distclean
distclean: clean
	echo -n 'Deleting *.bin and *.hex ... '
	rm -f *.bin *.hex
	echo 'done'

```
Add the `distclean` target.